apps/meteor/server/services/team/service.ts

```diff
@@ -77,6 +77,25 @@ export class TeamService extends ServiceClassInternal implements ITeamService {
 		};
 
 		try {
+			let roomId = room.id;
+			if (!roomId) {
+				const roomType: IRoom['t'] = team.type === TEAM_TYPE.PRIVATE ? 'p' : 'c';
+
+				const newRoom = {
+					...room,
+					type: roomType,
+					name: team.name,
+					members: memberUsernames as string[],
+					extraData: {
+						...room.extraData,
+					},
+					sidepanel,
+				};
+
+				const createdRoom = await Room.create(owner || uid, newRoom);
+				roomId = createdRoom._id;
+			}
```

Comment:
I know that you just moved the code, but here is the tip:
in that cases stay away from `let`, and also there is no reason to create extra `const` if they are going to be used only once.